### PR TITLE
Introduce lint base to make lint preset creation much easier

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: docs/presets/neutrino-lint-base/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+- Neutrino v4
+
+## Installation
+
+`neutrino-lint-base` can be installed via the Yarn or npm clients.
+
+#### Yarn
+
+```bash
+❯ yarn add neutrino-lint-base
```
Comment (helfi92) [inline docs/presets/neutrino-lint-base/README.md]: Should this be under dev dependencies?

Comment (eliperelman) [inline docs/presets/neutrino-lint-base/README.md]: No, the lint base is used in actual preset which is required by the dependent project. Extending from any preset needs it as a dependency.

Comment (eliperelman) [inline docs/presets/neutrino-lint-base/README.md]: The preset a **user is creating** will be installed in a project as a development dependency, the preset's dependency on another preset is a normal dep.

Comment (helfi92) [inline docs/presets/neutrino-lint-base/README.md]: Ah, OK, that makes sense.

**Pull request merged**